### ISSUE TITLE
[clusterfuzzlite] Remove unnecessary logs

### ISSUE THREAD

**Issue by jonathanmetzman** [google/oss-fuzz#6713]
Not doing this makes the output hard to read.

**Comment by jonathanmetzman** [comment]
@oliverchang I think this will require some libclusterfuzz changes, can you do this?

**Comment by jonathanmetzman** [comment]
I think this is particularly important for GCB where logs must be inspected to find the URL of the testcase.

**Comment by oliverchang** [comment]
I don't think we need to make any libClusterFuzz changes? The stacktrace is there already. 

That said, I'm not sure this helps the GCB case too much. If we upload this as an artifact, then users will have to find the URL to that in the first place?
